vignettes/development.Rmd

```diff
@@ -0,0 +1,49 @@
+---
+title: "Development"
+output: rmarkdown::html_vignette
+date: "`r Sys.Date()`"
+vignette: >
+  %\VignetteIndexEntry{Development}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+Notes on the RNAsum Rmd structure for developers
```

Comment:
Nice! Really handy.

Reply:
Those were copied from the Rmd itself, not taking any credit (will also need to be updated!)

Reply:
The comment was on the "Notes on the RNAsum Rmd...." bit not the content, as I found the idea to have it all in `development.Rmd` useful.